get.php

```diff
@@ -155,6 +155,7 @@
 	"vlc" => "https://github.com/javidominguez/VLC/releases/download/2.14/VLC-2.14.nvda-addon",
 	"vlc-dev" => "https://github.com/javidominguez/VLC/releases/download/2.13/VLC-2.13.nvda-addon",
 	"vent" => "Ventrilo-1.0-dev.nvda-addon",
+	"vocalizer" => "https://github.com/ruifontes/vocalizer_expressive_driver/releases/download/3.1.8/vocalizer_expressive_driver-3.1.8.nvda-addon",
```

Comment:
Aren't there drivers for Vocalizer classic (version 2/direct) as well? If yes, may I suggest changing the key a bit? If not and if Vocalizer Direct (version 2) will not be updated or no updates to be provided, then let's keep the key defined.